buildbot-host/buildbot-worker-windows-server-2019/ec2.pkr.hcl

```diff
@@ -38,6 +38,12 @@ source "amazon-ebs" "buildbot-worker-windows-server-2019" {
   instance_type    = "t3a.large"
   region           = var.buildbot_windows_server_2019_ec2_region
 
+  launch_block_device_mappings {
```

Comment:
I did this already in openvpn-build (to be pushed out), increasing disk size to 80GB.